src/styles/card.scss

```
@@ -0,0 +1,89 @@
+body {
+  margin: 0;
+  font-family: Roboto, sans-serif;
+  font-weight: 400;
+  font-size: 12px;
+}
+
+.card {
+  width: 200px;
+  height: 408px;
+  box-sizing: border-box;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+
```

also you can add `display: flex` for easier centering
```suggestion
  display: flex;
  flex-direction: column;
```

---

ok